### ISSUE TITLE
AAF Writer: Use supported otio values instead of relying on metadata

### ISSUE THREAD

**Issue by freesonluxo** [PixarAnimationStudios/OpenTimelineIO#438]
We expect the otio values to be correct, but they're not. We use the values in metadata instead

```
length = otio_gap.duration().value
length = otio_gap.metadata["AAF"]["Length"]
```
```
timecode_length = clip.media_reference.available_range.duration.value
timecode_length = clip.metadata["AAF"]["Length"]
```
```
timecode_start = otio_clip.media_reference.available_range.start_time.value
timecode_start = otio_clip.metadata["AAF"]["StartTime"]

```

**Comment by freesonluxo** [comment]
@shahbazk8194 look into it

**Comment by jminor** [comment]
Addressed by #460
